src/components/ContactTop/index.jsx

```diff
@@ -76,9 +107,26 @@ const ContactTop = () => {
               </div>
             </div>
           </div>
+          <button
+            onClick={() => {}} // here The function when click button
+            className={cn(
+              'btn',
+              'w-2/12',
+              'h-full',
+              'text-c100',
+              'text-sm',
+              'font-bold',
+              'bg-c300',
+              {
+                hidden: !haveBtn
```

Comment:
Super Nice!
However, given that we have react now, lets do things better than hiding elements with css 😉

React is a tree of components renders multiple time, the less components/html to render the better, so you can easily now use `haveBtn` prop to conditionally render (add)/remove components.

Bounce tip:
You can also have a look at Higher-Order Components  which will help you to use one base component to create two different components (one with haveBtn:true and other without)
`TopContact' and `TopContactWithBtn'